docs/ocis/identity-provider/oidc/flow.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Flow
+weight: 40
+geekdocRepo: https://github.com/owncloud/ocis
+geekdocEditPath: edit/master/docs/ocis/identity-provider/oidc
+geekdocFilePath: flow.md
+---
+
+In its default configuration, Infinite Scale supports three authentication methods as outlined on the [OIDC official site](https://openid.net/specs/openid-connect-core-1_0.html#rfc.section.3):
+1. Authorization Code Flow
+2. Implicit Flow
+3. Hybrid Flow
+
+To authenticate using OIDC, both `client_id` and `client_secret` are essential.
+Infinite Scale doesn't offer dynamic registration . We have to use one from the default owncloud client.
+By default, [owncloud clients](https://doc.owncloud.com/server/next/admin_manual/configuration/user/oidc/oidc.html#client-ids-secrets-and-redirect-uris) are:
+- Desktop
+- Android
+- iOS
+
+While selecting owncloud client for authentication, take note of specific limitations such as `Redirect URI`
+| Source | Redirect URI |
+|------|--------|
+|Android|oc://android.owncloud.com|
+|iOS|oc://ios.owncloud.com|
+|Desktop|http://127.0.0.1 <br> http://localhost |
+
+In this documentation, desktop client `client_id` and `client_secret` is being used.
+```bash
+client_id=xdXOt13JKxym1B1QcEncf2XDkLAexMBFwiT9j6EfhhHFJhs2KM9jbjTmf8JBXE69
+client_secret=UBntmLjC2yYCeHwsyj73Uwo9TAaecAetRwMw0xYcvNL9yRdLSUi0hUAHfvCHFeFh
+```
+
+# Authorization Code Flow
```

Comment:
we have to start with a level 2 heading, because the document title is already level 1.